src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx

```diff
@@ -114,11 +119,20 @@ const createStyles = (themeColors: ThemeColors) =>
       textTransform: 'uppercase',
       marginBottom: 12
     },
+    subheaderIcon: {
+      marginBottom: 12,
+      marginRight: 10
+    },
     task: {
       width: '100%',
       padding: 24,
       paddingTop: 0
     },
+    taskHeaderVerified: {
+      display: 'flex',
```

Comment:
FYI the default display in react-native is `flex` so this isn't needed

Also we have a `...flexRowCentered()` util that you can use if you want!